packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -385,6 +384,13 @@ class ThemeData with Diagnosticable {
       'This feature was deprecated after v2.6.0-11.0.pre.',
     )
     Brightness? primaryColorBrightness,
+    @Deprecated(
+      'No longer used by the framework, please remove any reference to it. '
+      'For more information, consult the migration guide at '
+      'https://flutter.dev/docs/release/breaking-changes/toggleable-active-color#migration-guide. '
```

Comment:
broken, has it yet to be published to the website?

Reply:
Not yet https://github.com/flutter/website/pull/6626